### ISSUE TITLE
Add CLI runner

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/js-mock-ipfs-pinning-service#1]
Usage at https://github.com/ipfs-shipyard/js-mock-ipfs-pinning-service#usage is async and makes it tricky  into a blocking CLI tool

Can we add CLI support so `npx git://github.com/ipfs-shipyard/js-mock-ipfs-pinning-service.git` starts mock on some local port?

(We need this if we want to replace [rb-pinning-service-api](https://github.com/ipfs-shipyard/rb-pinning-service-api) in go-ipfs tests)

**Comment by lidel** [comment]
ping @Gozala (we need this if we want to replace [rb-pinning-service-api](https://github.com/ipfs-shipyard/rb-pinning-service-api) in go-ipfs tests)

**Comment by Gozala** [comment]
Fixed by 84f6c1c now running following will start it:

```
npx git://github.com/ipfs-shipyard/js-mock-ipfs-pinning-service.git --port 8080 --token secret
```
